_posts/2017-05-02-newsletter.md

```diff
@@ -0,0 +1,156 @@
+---
+layout: newsletter
+title: "Newsletter #8 - Open up the Windows"
+category: newsletter
+permalink: /news/2017/05/
+---
+
+What is Neovim?
+---------------
+
+Neovim is a fork of the venerable text-editor vim, focused on extensibility and
+usability. It is not a rewrite but a continuation and extension of Vim. Many
+clones and derivatives exist, some very clever—but none are Vim. Neovim is built
+for users who want the good parts of Vim, and more. See a list of differences
+via [:help vim-differences](https://neovim.io/doc/user/vim_diff.html).
+
+### Participating
+
+If you are interested in contributing to Neovim, visit [the github
+page](https://github.com/neovim/neovim/blob/master/CONTRIBUTING.md) for details.
+We have a great test suite and automated CI, so you can just jump in and have a
+go. Don't be afraid to open a PR to get comments or just take advantage of the
+infrastructure!
+
+To get in touch with the team, join the [gitter channel](gitter.im/neovim) (also
+accessible via [IRC](https://webchat.freenode.net/?channels=neovim)), or visit
+the [Neovim subreddit](https://www.reddit.com/r/neovim/), which many of the
+developers read. Taking part in conversations is a contribution, too!
+
+
+What's new
+----------
+
+The [last newsletter](https://neovim.io/news/2016/11/) has gathered some dust,
+so we want to bring you up-to-date in the Neovim world. The elephant in the room
+is certainly
+
+### Release 0.2
+
+It features windows support, various terminal improvements, full API
+documentation for developers and a host of smaller fixes and improvements. A small highlight: A long standing issue with using `<Ctrl-h>` has been [fixed](https://github.com/neovim/neovim/wiki/FAQ#my-ctrl-h-mapping-doesnt-work).
+
+See
+the [release page](https://github.com/neovim/neovim/releases/tag/v0.2.0) for a
+detailed summary and download opportunities.
```

Comment:
This should really be condensed into the newsletter, that's what people want.

Reply:
Ah, I figured it's a thing of the past, and the newsletter should concentrate on the happenings after that :) No matter, I'll integrate it.